src/lib/apiTypesV2/Subscription.h

```diff
@@ -63,6 +65,8 @@ struct Notification
     blacklist(false),
     onlyChanged(false),
     timesSent(0),
+    counter(0),
+    maxLimit(0),
```

Comment:
`counter` and `maxLimit` are too broad names. I'd suggest to use some more specific ones, such as:

failsCounter
maxFailsLimit

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3890/commits/49b0c6df2c4d993b4e0fec160026fe0ac0017cb7